### ISSUE TITLE
which epoch can get the best result on test set?

### ISSUE THREAD

**Issue by wuqiangch** [clovaai/voxceleb_trainer#1]
@joonson ,Hi. The max epoch(=500)  is too large for me to train my dataset because of my k80 gpu. So  I want to know if the more epochs can get better result. Thanks!

**Comment by joonson** [comment]
![Picture1](https://user-images.githubusercontent.com/9700334/77823449-a0541f00-713e-11ea-8d3a-593b9d92b463.png)
![Picture2](https://user-images.githubusercontent.com/9700334/77823450-a21de280-713e-11ea-92e6-f677f2d189a1.png)

These are plots of EER vs epoch of a network using two different loss functions, tested using 2-second speech segments. The training takes 5 days on our GPU.
